.github/workflows/build-and-publish.yml

```diff
@@ -24,7 +22,20 @@ jobs:
   build:
     strategy:
       matrix:
-        version: [13, 14, 15, 16]
+        postgres_version: [13, 14, 15, 16]
+        major: [4,5]
+        include:
+          - partman_version: v4.7.4
+            partman_checksum: 28e4fdb83ecc16525959ae9593bfbfd077db429285f5f7d8e2468bfff6cbdbf2c81ace79a9ddeb4f00f51eb709163dbd713fe6b221a432ac2ff766f98d4cf8e4
+            default: "true"
+            major: 4
+          - partman_version: v5.0.0
+            partman_checksum: 7a1b41732eb063a94cee7a589a252055e3a33ccf0ddfe5cddba22978ffd306cf263312b172355c930d355aea0f5f5530fd2344520791df9cb05a38614c521f9f
+            default: "false"
+            major: 5
+        exclude:
+          - major: 5
+            postgres_version: 13 # partman 5.x does not support postgres 13
```

Comment:
Is it really necessary to include postgres 13, if it isn't supported by partman? The image actually promises to include partman in its name.

Reply:
You misunderstand. Partman 4.x supports Postgres 13, and right now, we offer an image for that. I wanted to keep that image supported. This exclude will prevent the matrix to run for the combination major:5 and postgres 13, so that it does not try to build an image with postgres 13 and partman 5.x, which would not be supported.

Reply:
You can actually see it nicely in the actions of this PR:
https://github.com/dbsystel/postgresql-partman-container/pull/12/checks
The first version is postgres, the second partman, and (13,5) is not in there.
![Screenshot 2023-10-27 at 09 22 08](https://github.com/dbsystel/postgresql-partman-container/assets/475516/ba922f32-3da3-4756-9706-e11c67d75afa)

Reply:
Ok, thanks!